newt/downloader/downloader.go

```diff
@@ -667,8 +667,8 @@ func (ld *LocalDownloader) CurrentBranch(path string) (string, error) {
 }
 
 func (ld *LocalDownloader) UpdateRepo(path string, branchName string) error {
-	// Nothing to update in a local repo.
-	return nil
+	os.RemoveAll(path)
```

Comment:
Hmmm, if I understand this correctly, it would be completely removing the local path including any changes that the user might have made, is that ok?

Reply:
It isn't ideal.  Ideally, newt would detect changes by comparing all files against the local repo's path.

The `local` repo type is just a debug feature, so I was a bit sloppy here.  I think a fix for this might be good as a separate PR.  What do you think?

Reply:
Yes, that would be ok... I just pointed out because I was not sure, thanks!